packages/sales-api-service/src/services/permissions.service.js

```diff
@@ -38,8 +38,7 @@ export const generatePermissionNumber = async (
   const initials = (firstName.charAt(0) + lastName.charAt(0)).toUpperCase()
   const block2 = permit.numberOfRods + channel + type + duration + age + initials
 
-  const block3 = cryptoRandomString({ length: 6, characters: 'ABCDEFGHJKLMNPQRSTUVWXYZ0123456789' })
-
+  const block3 = cryptoRandomString({ length: 6, characters: 'BCDFGHJKLMNPQRSTVWXZ256789' })
```

Comment:
Is it worth tightening up the permission number validator to reflect this? The current validator describes a superset of the unique address space

Reply:
Good point.  The API doesn't actually currently use the validator to actually do any validation - it is just used to form the swagger model to describe the response.  The current examples will now be technically invalid though so definitely something to look into.

Reply:
I've updated the validators but have actually relaxed the permission number validators to allow for old-style permission numbers otherwise users would have been unable to renew licences.